applications/ShapeOptimizationApplication/tests/algorithm_penalized_projection_test/parameters.json

```diff
@@ -62,14 +62,14 @@
         },
         "optimization_algorithm" : {
             "name"                    : "penalized_projection",
-            "correction_scaling"      : 1.0,
+            "correction_scaling"      : 0.2,
```

Comment:
Why did you change this? It is not related to the adaptive step_size right?

Reply:
No it's not. But I figured out, that in the penalized projection test, the scaling never actually was was adjusted, as the maximum scale of 1 is specified from the very beginning. So I reduced it a bit, to also include the constraint scaling into the testing process.